packages/ui/organisms/Hero/Hero.twig

```diff
@@ -0,0 +1,140 @@
+ *   customize the description element attributes.
+ * @param array                                     $ctas_attr
+ *   customize the cta element attributes.
+ */
+#}
+
+{% set content_position = content_position|default('center') %}
+
+{% set attributes =
+  merge_html_attributes(
+    attr ?? null,
+    { class: 'w-full min-h-screen' },
+    {
+      class: [
+        'hero flex relative',
+        {
+          'justify-start': content_position == 'left',
+          'justify-center text-center': content_position == 'center',
+          'justify-end': content_position == 'right'
+        }
+      ]
+    }
+  )
+%}
+
+{# Default images attributes #}
+{% set image_attributes =
+  merge_html_attributes(image_attr ?? null, { class: ['hero__image absolute inset-0'] })
+%}
+
+{# Default content attributes #}
+{% set content_attributes =
+  merge_html_attributes(
+    content_attr ?? null,
+    { class: 'flex flex-col justify-center h-screen w-1/2 gap-8' },
+    {
+      class: [
+        'hero__content relative',
+        {
+          'items-start': content_position == 'left',
+          'items-center text-center': content_position == 'center',
+          'items-end': content_position == 'right'
+        }
+      ]
+    }
+  )
+%}
+{% set surtitle_tag = title_tag|default('p') %}
+{% set surtitle_attributes =
+  merge_html_attributes(surtitle_attr ?? null, { class: ['hero__surtitle type-base'] })
+%}
+
+{% set title_tag = title_tag|default('p') %}
+{% set title_attributes =
+  merge_html_attributes(title_attr ?? null, { class: ['hero__title type-h1'] })
+%}
+
+{% set description_tag = description_tag|default('p') %}
+{% set description_attributes =
+  merge_html_attributes(description_attr ?? null, { class: ['hero__description type-base'] })
+%}
+
+{% set ctas_attributes =
+  merge_html_attributes(ctas_attr ?? null, { class: ['hero__ctas', 'flex', 'gap-6'] })
+%}
+
+<section {{ html_attributes(attributes) }}>
+  {% block background %}
+    {% if image %}
+      <div {{ html_attributes(image_attributes) }}>
+        {% include '@ui/atoms/Figure/Figure.twig' with image only %}
+      </div>
+    {% endif %}
+  {% endblock %}
+  {% block content %}
+    <div {{ html_attributes(content_attributes) }}>
+      {% block surtitle %}
+        {% if surtitle %}
+          {% html_element surtitle_tag with surtitle_attributes %}
+            {{ surtitle }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block title %}
+        {% if title %}
+          {% html_element title_tag with title_attributes %}
+            {{ title }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block description %}
+        {% if description %}
+          {% html_element description_tag with description_attributes %}
+            {{ description }}
+          {% end_html_element %}
+        {% endif %}
+      {% endblock %}
+      {% block ctas %}
```

Comment:
**Question (non-blocking):** What about renaming `ctas` to `actions` ?

Reply:
It is indeed easier to understand, good idea! 

Any thought @Liax and @perruche?

Reply:
yes `actions` is a clearer naming 👍

Reply:
The `ImageText` block also use the cta naming, can we update it as well ?

Reply:
Yes, it does make it more understandable